### PR TITLE
java class

### PR DESCRIPTION
**Pull request by sumit-03**

### CODE REVIEW

**Comment by sumit-03** [comment]
This is the OpenCV library(module).

File: .idea/gradle.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/openCVLibrary341" />
```
Comment (Zoham) [inline .idea/gradle.xml:12]: do not include files from .idea folder

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zoham) [inline .idea/runConfigurations.xml:1]: same here - no .idea folder changes

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zoham) [inline .idea/misc.xml:1]: same here - no .idea folder changes

**Comment by Zoham** [comment]
OpenCV Library files not included in commit
Check the above comments
use this file for HandleCamera https://pastebin.com/zzHdivCG
rest seems fine 
make changes accordingly

**Comment by sumit-03** [comment]
did u see the pull request having heading java class for handling camera it has same code as in the link.
and also I commited the open Cv library.

**Review by Zoham — APPROVED** [review]

**Pull request merged**